life_expectancy/cleaning.py

```diff
@@ -18,32 +25,87 @@ def clean_data(region_code:str, data_to_clean:DataFrame):
         origin and only containing data from the region selected
     """
 
+    data_to_clean = data_to_clean.rename(columns={data_to_clean.columns[3]: "region"})
+    data_to_clean = data_to_clean.rename(columns={data_to_clean.columns[5]: "value"})
+
+    cleaned_data = data_to_clean.dropna()
+    data = cleaned_data[cleaned_data['region'] == str(region_code)]
+    return data
+
+def clean_data_tsv(region_code:Country, data_to_clean:DataFrame):
+    """This function receives data from the original .tsv file and cleans it for further
+    usage.
+    params:
+        data_to_clean: pandas Dataframe - This is the data obtained from the .tsv file
+            uncleaned and containing information from all regions and subjects.
+        region_code: string - This is the country code for selecting the information
+            wanted for usage. ex: PT for Portugal
+    return: data: Pandas Datafrane - final dataframe containing the clean data from the
+        origin and only containing data from the region selected
+    """
     data_to_clean = data_to_clean.rename(columns={data_to_clean.columns[3]: "region"})
     data_to_clean = pd.melt(data_to_clean,id_vars=['unit','sex','age',"region"],var_name='year')
     data_to_clean = data_to_clean.astype({'year':'int'})
     data_to_clean['value'] = pd.to_numeric(data_to_clean.value.str.replace(r'[^\d.]',''),
-                                           errors='coerce')
+                                               errors='coerce')
+
     cleaned_data = data_to_clean.dropna()
-    data = cleaned_data[cleaned_data['region'] == region_code]
+    data = cleaned_data[cleaned_data['region'] == str(region_code)]
```

Comment:
```suggestion
    data = cleaned_data[cleaned_data['region'] == region_code.name]
```

**Suggestion**: This is just a suggestion. Your approach is not wrong in any way, I just found it a bit weird that we are giving the method this enum and then transforming it back to a string instead of using its attributes